.mailmap

```diff
@@ -537,7 +537,7 @@ Sangdi Xu <xu.sangdi@h3c.com>
 Sangdi Xu <xu.sangdi@h3c.com> <xu.sangdi@h3c>
 Scott A. Brandt <scott@cs.ucsc.edu> <sbrandt@29311d96-e01e-0410-9327-a35deaab8ce9>
 Sebastian Wagner <swagner@suse.com> <sebastian.wagner@suse.com>
-Sebastian Wagner <swagner@suse.com> <sebastian@spawnhost.de>
+Sebastian Wagner <swagner@redhat.com> <sebastian@spawnhost.de>
```

Comment:
unsure about this one. I really tried hard to avoid signing anything with my private email. But seems it ended up here anyway.

Reply:
neither am i.. guess it's up to you =)